src/main/resources/db/migration/R__grant_to_cloudsqliamuser.sql

```diff
@@ -1,4 +1,4 @@
 GRANT SELECT ON ALL TABLES IN SCHEMA public TO cloudsqliamuser;
 GRANT SELECT ON ALL TABLES IN SCHEMA public TO "isyfo-analyse";
 -- GRANT SELECT, INSERT, UPDATE, DELETE ON ALL TABLES IN SCHEMA public TO cloudsqliamuser;
--- GRANT ALL ON ALL SEQUENCES IN SCHEMA public TO cloudsqliamuser;
+-- GRANT ALL ON ALL SEQUENCES IN SCHEMA public TO cloudsqliamuser; --
```

Comment:
Det er en liten endring i dette scriptet for å lure flyway til å kjøre scriptet på nytt. Vi trenger at den kjører slik at vi får tilgang til å gjøre select på den nye tabellen.

Reply:
Her mangler faktisk det som bør være første linje i scriptet: 
REVOKE ALL ON ALL TABLES IN SCHEMA public FROM cloudsqliamuser;

Så kan jo bruke anledningen til å legge inn det.